README.md

```diff
@@ -31,3 +31,46 @@ Because all efforts have shifted to PhpSpreadsheet, PHPExcel will no longer be m
 ## License
 
 PhpSpreadsheet is licensed under [LGPL (GNU LESSER GENERAL PUBLIC LICENSE)](https://github.com/PHPOffice/PhpSpreadsheet/blob/master/LICENSE)
+
+### TODO
```

Comment:
This file should be kept as short as possible. Instead we have a page in documentation that cover supported functions. That should be where we keep a complete list of supported/non-supported functions